src/libraries/Microsoft.PowerFx.Connectors/Execution/HttpFunctionInvoker.cs

```diff
@@ -55,6 +58,7 @@ public HttpRequestMessage BuildRequest(FormulaValue[] args)
         {
             var path = _path;
             var query = new StringBuilder();
+            var headers = new Dictionary<string, string>();
```

Comment:
>headers [](http://example.com/codeflow?start=16&length=7)

case sensitive or insensitive?
(you can specify via a parameter to the ctor)

Reply:
Fixed